test/lib/mayaUsd/render/vp2RenderDelegate/testVP2RenderDelegateUsdCamera.py

```diff
@@ -107,6 +107,9 @@ def testStaticUsdCamera(self):
         clippingAttr = usdCamera.GetAttribute('clippingRange')
         clippingRange = clippingAttr.Get()
         clippingAttr.Set(Gf.Vec2f(10, 5000))
+        # Translate the camera to force the view to update
```

Comment:
This seems a bit hacked. But if it's unblocking us for the time being I think it should be fine.